pom.xml

```diff
@@ -25,13 +25,14 @@
     <junit-version>4.13.2</junit-version>
     <mockito-version>4.8.1</mockito-version>
     <activemq-version>5.18.1</activemq-version>
+    <activemq-snapshot-version>5.18.1.2</activemq-snapshot-version>
```

Comment:
This PR will not build against a standard activemq build. We can't add arbitrary code that will not work with an upstream.